modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -98,6 +98,8 @@
 				[codeblock]
 				[{function:bar, line:12, source:res://script.gd}, {function:foo, line:9, source:res://script.gd}, {function:_ready, line:6, source:res://script.gd}]
 				[/codeblock]
+
+				Note: Not supported for calling from threads. Instead, this will return an empty array.
```

Comment:
Two things to change:
- No empty lines (one line == one paragraph)
- "Note:" should be formatted like this: `[b]Note:[/b]` for consistency with the rest of the documentation.

Reply:
Thanks, fixed